src/main/java/me/botsko/prism/utils/BlockUtils.java

```diff
@@ -122,4 +122,10 @@ public static int blockIdMustRecordAs(int block_id) {
         final Material[] materials = { Material.WATER, Material.STATIONARY_WATER };
         return removeMaterialsFromRadius( materials, loc, radius );
     }
+
+    public static boolean isDoor(Material m) {
```

Comment:
I reverted it - I assume we had it as we hadnt used the elixir import at that stage...I have removed this